### ISSUE TITLE
The page progress is not accurate.

### ISSUE THREAD

**Issue by JowelTisso** [farshed/duofolio#18]
@farshed @rafaelsaback @threenplusone  The page progress is not accurate. I am able to figure out why. It may be because the actually single page is divided in 2 or 3 pages so the page progress increments after 2 or 3 swipes.
I have an ebook which shows 46  total pages but counting with swipes it has 58 pages.
Please fix this issue or direct me.

**Comment by PekopT** [comment]
hi, @JowelTisso!! Have you fixed this problem ?

**Comment by JowelTisso** [comment]
@PekopT No, I haven't. I have just ignored the page count with progress slider... I guess the issue is in epubjs, when injecting javascript in webview.
